test/integration/good/overloading/pretty.expected

```diff
@@ -214,13 +214,20 @@ functions {
     return dydt;
   }
   
-  vector simple_SIR(real t, vector y, real beta,
+  vector simple_SIR(real t,
+                    vector y,
+                    real beta,
                     // water contact rate
-                    real kappa, // C_{50}
-                    real gamma, // recovery rate
-                    real xi, // bacteria production rate
-                    real delta, // bacteria removal rate
-                    int unused // just for testing
+                    real kappa,
+                    // C_{50}
+                    real gamma,
+                    // recovery rate
+                    real xi,
+                    // bacteria production rate
+                    real delta,
+                    // bacteria removal rate
+                    int unused
+                    // just for testing
```

Comment:
This is rather unfortunate. Makes it look like the comment applies to next item.

Reply:
I managed to get them back on the same line as the commas (it still doesn't know what the user intended, but at least it preserves previous behavior)